ui/src/error.ts

```diff
@@ -5,12 +5,20 @@
 // LICENSE file.
 
 import * as svelteStore from "svelte/store";
-import * as lodash from "lodash";
+import lodash from "lodash";
 
-import * as config from "ui/src/config";
-import * as notification from "./notification";
+import * as bacon from "./bacon";
 import * as ipc from "./ipc";
 
+// We use an event bus to break up the dependency cycle between the `error`,
+// `notification` and `browserStore` modules. This also makes the mocks in the
+// unit tests simpler.
```

Comment:
You can delete this comment. I don’t think we need to justify architecture and API design decisions. We should reserve these comments for code that does non-obvious and quirky things.

Reply:
3aeb9693966575f9184ff7e2ef291afd7edb4090